### PR TITLE
Added new domains to waaw resolver

### PR DESCRIPTION
**Pull request by MrDini123**
Hey there,

I noticed that there are a few domains that aren't handled yet so I added them.

### CODE REVIEW

**Comment by Gujal00** [comment]
Added in https://github.com/Gujal00/ResolveURL/commit/1b14ac586fcbdffe29ba568feb27d9f4eaaa5d02

File: script.module.resolveurl/lib/resolveurl/plugins/waaw.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 class WaawResolver(ResolveUrl):
     name = 'Waaw'
     domains = ['waaw.ac', 'netu.ac', 'hqq.ac', 'waaw.tv', 'netu.tv', 'hqq.to',
-               'doplay.store', 'younetu.com', 'stbnetu.xyz']
+               'doplay.store', 'stbnetu.xyz', 'waaw.to', 'netu.to', 'hqq.ac']
```
[the remark this answers is not in the document]
Comment (MrDini123) [reply script.module.resolveurl/lib/resolveurl/plugins/waaw.py:34]: True, my bad. Thanks for the fix.

**Comment by MrDini123** [comment]
Btw younetu.com expired, that's why I removed it from the list.

**Comment by Gujal00** [comment]
yes younetu.com expired last week. may be it will come back, so for now leave it in the resolver. If it still is down after a month we can remove it

**Pull request closed without merging**